examples/ios_app/Utils/BUILD

```diff
@@ -1,3 +1,5 @@
+load("@rules_cc//cc:defs.bzl", "objc_library")
```

Comment:
I don't want to depend on `rules_cc` as it's actually bad to use it right now, until it starts to get updated with recent Starlarkification efforts.

Reply:
Buildifier added it. I am looking into how we can avoid this being added in the future.

Reply:
I think that I understand how this appeared now. I run Buildifier in my editor with the flag `-warnings all`. That added the load statement. The Buildifier check/fix that we added in a previous PR currently does not run with all of the warnings. I added #176 to track this. For now, I added the file with the `objc_library` to the exclude list.

Reply:
I just found a warning level that appears to apply to the load statement (`cc-native`). I will apply it in the PR for #176.